idaes/models/unit_models/tests/test_heat_exchanger_1D.py

```diff
@@ -3544,7 +3544,7 @@ def model(self):
 
         return m
 
-    @pytest.mark.component
+    @pytest.mark.integration
     def test_general_hx1d_initializer(self, model):
         initializer = HX1DInitializer()
         initializer.initialize(model.fs.unit)
```

Comment:
I wonder how this initialization time is going to be affected when we move away from block triangularization in #1436 .

Reply:
That is a good point, but I suspect it won't have a huge effect in this case. I believe there is another equivalent test for a different configuration that runs fast enough, so I suspect this one is caused more by stiffness in the model than the property initialization. Still, would be worth waiting for #1436 to check.

Reply:
Update: after #1436 merged, run time for this test is still ~16 seconds, so I'll leave it in integration for now.